ports/geckolib/glue.rs

```diff
@@ -2216,14 +2214,14 @@ fn fill_in_missing_keyframe_values(all_properties:  &[TransitionProperty],
         return;
     }
 
-    let keyframe = match offset {
-        0. => unsafe {
+    let keyframe = if is_initial {
```

Comment:
@emilio never, never, _never_ compare float values for equality if you can help it.

And unreachable branches are not great.